gtsfm/averaging/translation/averaging_1dsfm.py

```diff
@@ -140,11 +150,91 @@ def compute_inlier_mask(
 
         return inlier_idxs
 
+    def __get_priors_hilti_rig(
+        self,
+        i2Ti1_priors: Optional[Dict[Tuple[int, int], PosePrior]],
+        wRi_list: List[Optional[Rot3]],
+    ) -> gtsam.BinaryMeasurementsPoint3:
+        """Converts the priors from relative Pose3 priors to relative Point3 priors in world frame.
+        
+        If the priors are hard constraints (in the same rig), a hard-coded noise model is used. 
+        If the priors are soft constraints, the covariance from the PosePrior is used.
+
+        Soft constraints are only added between the 3rd rig cameras. 
+        Hard constraints are only added between the 3rd camera and other cameras in same rig.
+
+        Args: 
+            i2Ti1_priors: Relative pose priors between cameras, could be a hard or soft prior.
+            wRi_list: Absolute rotation estimates from Shonan averaging.
```

Comment:
We should drop Shonan averaging and instead say rotation averaging.

Reply:
Done